source/docs/tuner/device-list.rst

```diff
@@ -76,8 +76,12 @@ Once the upgrade dialog is opened, information detailing the device name, model,
 
 .. note:: The :guilabel:`Pro` column has no effect if you're using the update offline button.
 
-The user can begin the upgrade progress by selecting :guilabel:`Upgrade Online (latest)` or :guilabel:`Upgrade Offline (file)`. The first option will upgrade all listed devices to their latest available firmware (Pro or Phoenix v5 depending on the toggle state), downloading firmware if they do not already exist in cache. The latter option will prompt for each firmware file.
+The user can begin the upgrade progress by selecting :guilabel:`Upgrade to latest` or :guilabel:`Select firmware...`. The first option will upgrade all listed devices to their latest available firmware (Pro or Phoenix v5 depending on the toggle state). The latter option will open a popup allowing you to select a specific version or firmware per model.
```

Comment:
"allowing you to select a specific version or firmware file per model." (add "file")